packages/hooks/src/useDepositFiatAddress.ts

```diff
@@ -6,8 +6,9 @@ const useDepositFiatAddress = () => {
     const { ui } = useStore();
     const { is_dark_mode_on } = ui;
     const { data, mutate, ...rest } = useRequest('cashier');
+    const should_enable_dark_mode = is_dark_mode_on ? 'on' : 'off';
```

Comment:
```suggestion
    const dark_mode = is_dark_mode_on ? 'on' : 'off';
```
It's not a boolean.

Reply:
Oooh got it I didn't knew about this, I'll use `should_*` for boolean vars from now on 😓

Reply:
@adrienne-deriv It's a common best practice to name boolean variables with `is`, `has`, or `should` so it's easier to identify a boolean variable without checking its value or type 🙇🏻